h2/src/main/org/h2/mode/FunctionsMySQL.java

```diff
@@ -131,9 +130,13 @@ public static String fromUnixTime(int seconds) {
      * @param format The format of the date/time String to return.
      * @return a formatted date/time String in the given format.
      */
-    public static String fromUnixTime(int seconds, String format) {
+    public static String fromUnixTime(SessionLocal session, int seconds, String format) {
         format = convertToSimpleDateFormat(format);
         SimpleDateFormat formatter = new SimpleDateFormat(format, Locale.ENGLISH);
```

Comment:
`SimpleDateFormat` and `Date` are legacy defective by design classes, please, replace them with classes from JSR-310.
Something like
```Java
DateTimeFormatter.ofPattern(format).withZone(ZoneId.of(session.currentTimeZone().getId()))
        .format(Instant.ofEpochSecond(seconds));
```

You also need to adjust tests it `TestCompatibility`, they must set some specific time zone of the session.